src/VisualStudio/Core/Def/Implementation/VirtualMemoryNotificationListener.cs

```diff
@@ -83,6 +83,8 @@ public int OnBroadcastMessage(uint msg, IntPtr wParam, IntPtr lParam)
             {
                 case VSConstants.VSM_VIRTUALMEMORYLOW:
                 case VSConstants.VSM_VIRTUALMEMORYCRITICAL:
+                case VSConstants.VSM_MEMORYHIGH:
```

Comment:
Is there any docs on what these values are? Would be interested in seeing how we can implement something similar in VSMac (we already get memory pressure events from the OS)